src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -371,6 +371,11 @@ internal void InsertGraph()
         internal delegate void InsertDocumentationLinkEventHandler(object sender, InsertDocumentationLinkEventArgs e);
         internal event InsertDocumentationLinkEventHandler HandleInsertFile;
 
+        private string DynamoGraphFromMDFilePath(string path)
+        {
+            return path.Replace(".md", ".dyn").Replace("%20", " ");
```

Comment:
You can use Path.GetFileNameWithoutExtension to trim the extension name and add .dyn. That would be robust.

Reply:
Yup, great idea! Done.